cmd/openshift-install/create.go

```diff
@@ -96,6 +96,10 @@ var (
 
 				err = waitForBootstrapComplete(ctx, config, rootOpts.dir)
 				if err != nil {
+					err2 := runGatherBootstrapCmd(nil, nil)
+					if err2 != nil && err2 != context.Canceled {
```

Comment:
why the `context.Canceled`

Reply:
@abhinavdahiya I forgot to delete it from a copy/paste. Modified, ptal.